src/en/getting-started.md

```diff
@@ -131,12 +131,18 @@ required to configure IPv6 networking).
    
 LXD is now configured to work with Juju.
 
+!!! Note: LXD adds iptables rules to enable traffic to the subnet/bridge it
+created. If you subsequently add/change firewall settings (e.g. `ufw`), you
+should ensure that these rules still work for Juju to function properly.
```

Comment:
This wording is awkward. Maybe: "If you subsequently add/change firewall rules (e.g. `ufw`), ensure that they do not prevent Juju from communicating with LXD."